### DISCUSSION TITLE
HitObject may not be visible when dragging the progress bar quickly

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#22314] [Q&A] [answered]
https://user-images.githubusercontent.com/74862110/213864090-80a88b98-4b40-45fd-8c0c-5901e2751ebf.mp4

**Comment by Jan-Arbus** [comment]
This issue can only be reproduced in Argon skin
2023.121.0 can still reproduce this bug

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/21089
